consensus/src/epoch_manager.rs

```diff
@@ -351,6 +362,9 @@ impl<T: Payload> EpochManager<T> {
         ledger_recovery_data: LedgerRecoveryData,
         epoch_info: EpochInfo,
     ) {
+        // @REVIEW: might it be necessary to update the network config as sync happens?
+        // What if we are so far out of date that the network info is invalid? Is there a way
+        // to bootstrap?
```

Comment:
I believe the solution for V1 is to restore backup to a state that's relatively new (have at least one peer)